src/lib.rs

```diff
@@ -551,6 +552,9 @@ impl CpuId {
     }
 
     /// Information about topology (LEAF=0x0B).
+    /// 
+    /// Intel SDM suggests software should check support for leaf 0x1F ([`crate::get_extended_topology_info_v2()`]), and
```

Comment:
I think this should be [`CpuId::get_extended_topology_info_v2()`] instead?

Reply:
Agreed, I'll resolve this.